config/frontend.example.toml

```
@@ -61,6 +61,11 @@ runtime_size = 8
 ## - `all`: enable all compression.
 ## Default to `none`
 flight_compression = "arrow_ipc"
+## The maximum connection age for gRPC connection.
+## The value can be a human-readable time string. For example: `10m` for ten minutes or `1h` for one hour.
+## Refer to https://grpc.io/docs/guides/keepalive/ for more details.
+## @toml2docs:none-default
+max_connection_age = "365d"
```

So users don't set the max_connection_age if they use the config to start the server.
```suggestion
#+ max_connection_age = "10m"
```